app/lib/steps/capital_section.rb

```diff
@@ -2,7 +2,7 @@ module Steps
   class CapitalSection
     PROPERTY_STEPS = %i[property property_entry].freeze
     VEHICLE_STEPS = %i[vehicle vehicle_details vehicles_details].freeze
-    TAIL_STEPS = %i[assets].freeze
+    TAIL_STEPS = %i[assets partner_assets].freeze
```

Comment:
Would it make sense to rename this ASSET_STEPS, given that in the new flow it doesn't appear on the tail end, it's at the start, and we don't use this constant in the old flow at all?